WORKSPACE

```diff
@@ -28,7 +28,34 @@ load(
 haskell_nixpkgs_package(
     name = "ghc",
     attribute_path = "haskellPackages.ghc",
-    build_file = "//haskell:ghc.BUILD",
```

Comment:
Does this mean that `//haskell:ghc.BUILD` is now only used for `ghc_bindist`? Shouldn't the name change?

Reply:
With the latest commit in the series, we no longer need to duplicate much of the content of this file. So we're back to using `//haskell:ghc.BUILD` here as before.